pkg/nrtupdater/nrtupdater.go

```diff
@@ -15,6 +15,15 @@ import (
 	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/dumpobject"
 )
 
+const (
+	AnnotationRTEUpdate = "k8stopoawareschedwg/rte-update"
+)
+
+const (
+	RTEUpdatePeriodic = "periodic"
+	RTEUpdateReactive = "reactive"
+)
+
 var (
```

Comment:
There was feedback on the NFD PR to switch from using log to klog, looks like there is an inclination towards usingklog and we should consider doing that in RTE as well. Could be a completely separate PR.

Reply:
ack, let's address indeed.